version/version.go

```diff
@@ -17,5 +17,5 @@ func FormattedVersion() string {
 	if Version == "" {
 		return "unknown"
 	}
-	return fmt.Sprintf("%s (%s @ %s)\n", Version, BuildVersion, BuildTime)
+	return fmt.Sprintf("%s (%s @ %s)", Version, BuildVersion, BuildTime)
```

Comment:
Nope, all good. There was actually 1 too many new lines there, so this actually fixed that. :)